### ISSUE TITLE
Release more regularly

### ISSUE THREAD

**Issue by nitind** [eclipse/buildship#1225]
The most recent release of Buildship was before the middle of last year, the one before that over a year earlier. I see that commits are going in during most months, so could the release cadence be increased so that there are fewer issues integrating Buildship into quarterly releases--particularly since the Platform decided to pull its dependencies from Maven directly and leave the rest of the ecosystem to keep up or cause wiring problems?

**Comment by tdelhomenie-microej** [comment]
I agree, that's a pity that bugs fixed for months are still not available to end users. It should be released more frequently, even if each release contains only very few bugs fixes. Is there a way to help on this?

**Comment by erizzo** [comment]
Are any of the developers paying attention to the GitHub discussions? Seems not; this one and [the one about Undo/Redo being broken](https://github.com/eclipse/buildship/issues/1188) are completely ignored.
@waynebeaton sorry for the out-of-the-blue tag, but it's been a while since I was involved at this level - how do we assess the level of community involvement from a project's team these days?
